src/app/pages/home/containers/home-page.component.html

```diff
@@ -26,15 +26,32 @@
   </div>
 
   <div
-    *ngIf="hasClinicalTasks | async"
-    class="clinical-container"
-    (tap)="openClinicalTasksPage()"
+    class="clinical-container clinical-button"
+    [class.clinical-button-bottom]="showCalendar"
   >
-    <div class="clinical-button" [class.clinical-button-bottom]="showCalendar">
-      <div class="clinical-icon">
-        <img src="assets/imgs/clinical.png" class="icon-clinical" />
-      </div>
-    </div>
+    <ion-fab left bottom>
+      <button ion-fab class="fab">
+        <ion-icon name="arrow-dropup"></ion-icon>
```

Comment:
Maybe the icon can be a `+`? Makes it more obvious that we are creating a new response.

Reply:
Made the changes. I've updated the screenshot as well.